scitran/data/medimg/dcm/dcm.py

```diff
@@ -235,20 +235,26 @@ def __init__(self, path, load_data=False, timezone=None):
 
         """
         super(Dicom, self).__init__(path, load_data, timezone)
-        with tarfile.open(path) as archive:
-            for ti in archive:
+        zip_dicom = zipfile.ZipFile(path)
+        for filename in zip_dicom.namelist():
+            try:
+                zip_content = zip_dicom.open(filename)
+                self._hdr = MetaExtractor(dicom.read_file(cStringIO.StringIO(zip_content.read()), stop_before_pixels=True))
```

Comment:
With tarfile, we needed the StringIO wrapper to improve performance. Is that still needed with zipfile?